SoftServeCarMobile/src/activity/messages/messages-tabs/MessagesTabs.tsx

```diff
@@ -19,6 +19,11 @@ const ChatTabs = () => {
     const userServices = container.resolve(UserService);
     const [currentUser, setCurrentUser] = useState({} as User);
     const { user } = useContext(AuthContext);
+    const [isOpenFilter, setIsOpenFilter] = useState(false);
+
+    const setIsOpen = () => {
+        setIsOpenFilter(!isOpenFilter);
```

Comment:
we set isOpenFilter, setIsOpenFilter to false, and after that revert the value. Is it better to set the default state to true?

Reply:
I'll explain to you why I've written it in such a way in the daily.